### PR TITLE
Add interesting tech in bitcoin post

### PR DESCRIPTION
**Pull request by leroix**
[Rendered post](https://github.com/verypossible/engineering-blog/blob/interesting-bitcoin/content/post/interesting-tech-in-bitcoin.md)

### CODE REVIEW

File: content/post/interesting-tech-in-bitcoin.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+In Bitcoin, when we want to spend an unspent transaction output that was addressed to a hash of our public key, we sign it using our private key and provide our public key. Other nodes can check that the signature is correct, meaning the spend is valid, given the unspent transaction output details and the public key.
+
+## Elliptic Curve Cryptography
+For the public and private keys needed to establish bitcoin addresses and allow a user to spend an unspent transaction output by signing it, Bitcoin uses elliptic curve cryptography. Elliptic curve cryptography was chosen for Bitcoin over RSA because it offers better security for a fraction of the storage space. Elliptic curve cryptography takes advantage of a strange symmetry that exists in a class of mathematical curves defined by
```
Comment (bbrock25) [inline content/post/interesting-tech-in-bitcoin.md]: I feel that this first sentence is hard to follow.

Comment (leroix) [reply content/post/interesting-tech-in-bitcoin.md]: Hehe, yea I agree. I fix that.
